app/services/idv/socure_user_set.rb

```diff
@@ -9,11 +9,12 @@ def initialize(redis_pool: REDIS_POOL)
     end
 
     def add_user!(user_uuid:)
-      return if maxed_users?
+      return false if maxed_users?
 
       redis_pool.with do |client|
         client.sadd(key, user_uuid)
       end
+      true
```

Comment:
btw, looks like [client.sadd returns a count of elements added](https://github.com/redis/redis-rb/blob/b6fd17b28be94cb497d76c9394676d8317e941d5/lib/redis/commands/sets.rb#L18) that could be used to know if the user really was added
👀